app/src/main/res/layout/fragment_dashboard.xml

```diff
@@ -21,14 +21,35 @@
         android:layout_height="match_parent"
         android:background="@color/grey_background">
 
-    <TextView
-            android:id="@+id/tvNoWorkflow"
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_centerVertical="true"
+        android:layout_centerHorizontal="true"
+        android:visibility="gone"
+        android:id="@+id/tvNoWorkflow"
+        android:layout_marginBottom="@dimen/relative_layout_margin_bottom">
+
+        <ImageView
+            android:id="@+id/workflow_icon"
```

Comment:
use camel case for id name

Reply:
Hey @sagar15795 , id naming should not be in camel case according to https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md#2321-id-naming

Reply:
use proper prefix.